Source/Core/DolphinQt2/Config/Graphics/GraphicsWidget.h

```diff
@@ -0,0 +1,39 @@
+// Copyright 2017 Dolphin Emulator Project
+// Licensed under GPLv2+
+// Refer to the license.txt file included.
+
+#pragma once
+
+#include <QHash>
+#include <QWidget>
+
+class QFormLayout;
+class QGroupBox;
+class QLabel;
+
+class GraphicsWidget : public QWidget
```

Comment:
What I envisioned was the description box as its own class, that each tab can add on the bottom (or the `GraphicsWindow` adds it to each, maybe?). Then this `GraphicsWidget` class would be removed entirely. Something like:

```c++
class ShowDescriptionsWhenHoveringOverOtherWidgetsBox : public QGroupBox
{
public:
  AddDescriptionForWidget(QWidget *widget, const char* description);
private:
  bool eventFilter(QObject* object, QEvent* event) override;
  QLabel* m_description;
  QHash<QObject*, const char*> m_widget_descriptions;
};
```

Can we remove this `GraphicsWidget` class entirely? The form layout shouldn't be used here, and exposing the descriptions`DescriptionAdded` could be done on each tab individually (either adding the signal on each class, or, probably more simply, exposing `void AddDescriptions(widget_implementing_descriptions)`).

The current version you uploaded places the description box slightly differently (outside the tabs box, screenshot attached) from how DolphinWX does it (inside the tabs box). Is that intended?

<img width="724" alt="screen shot 2017-06-18 at 2 56 07 pm" src="https://user-images.githubusercontent.com/594093/27264575-5cbe3ce4-5436-11e7-939e-c27885e5da13.png">